src/mui/button/SaveButton.js

```diff
@@ -49,6 +52,12 @@ SaveButton.propTypes = {
     raised: PropTypes.bool,
     saving: PropTypes.bool,
     translate: PropTypes.func.isRequired,
+    submitOnEnter: PropTypes.bool,
+    handleSubmit: PropTypes.func,
+};
+
+SaveButton.defaultProps = {
+    submitOnEnter: true,
```

Comment:
I feel it is something that belongs more to the form than the button, but even than I will need to set it on <SimpleForm> and <TabbedForm> too. Is that okay with you?